# An Array of obstacles in DuDrive

Header header

DriveObs[]  obstacles

